administrator/templates/atum/css/template.css

```diff
@@ -762,6 +751,15 @@ iframe {
       text-decoration: none;
       outline: 0;
       box-shadow: inset 0 0 0 1px #78aeda; }
+  .main-nav li .collapse-arrow::after {
+    position: absolute;
+    top: 50%;
+    right: 15px;
+    margin-top: -.45rem;
```

Comment:
Instead of the negative margin-top could you use `transform: translateY(-50%);`

It will keep the caret vertically centered regardless of font-size